extension/dnsmasq/dnsmasq.js

```diff
@@ -692,9 +694,9 @@ module.exports = class DNSMASQ {
       if (type === "family") {
         targetIP = BLUE_HOLE_IP
       }
-
+      const tag = this.level[type] == "system" ? "" : "$ad_block";
```

Comment:
Do not hardcode ad_block here. You can use value of 'type' as tag, here type is 'adblock'.